src/sentry/static/sentry/app/views/groupActivity/noteInput.jsx

```diff
@@ -70,6 +70,11 @@ var NoteInput = React.createClass({
     this.setState({value: e.target.value});
   },
 
+  onKeyDown(e) {
+    // Auto submit the form on [meta] + Enter
+    e.keyCode === 13 && e.metaKey && this.onSubmit(e)
```

Comment:
Is there a better way to do this than calling `this.onSubmit(e)`? Seems kinda hacky since it's passing along the event object that wasn't intended for a submit.